uint8 cuboid=0
uint8 cylinder=1
uint8 sphere=2

uint8 type
float64 height
float64 width
float64 length
float64 radius
float64 pos_x
float64 pos_y
float64 pos_z
float64 rot_x
float64 rot_y
float64 rot_z
float64 rot_w
